service/src/main/java/de/adorsys/psd2/sandbox/xs2a/service/redirect/RedirectService.java

```diff
@@ -85,6 +98,26 @@ public void handleConsentCreationRedirectRequest(String externalId, String psuId
     aisConsentAuthorizationRepository.save(aisConsentAuth);
   }
 
+  private TppAccountAccess getTppAccountAccess(Account account, TypeAccess typeAccess) {
+    TppAccountAccess accountAccess = new TppAccountAccess();
+    accountAccess.setAccountIdentifier(account.getIban());
+    accountAccess.setCurrency(account.getCurrency());
+    accountAccess.setTypeAccess(typeAccess);
+    accountAccess.setAccountReferenceType(AccountReferenceType.IBAN);
+    return accountAccess;
+  }
+
+  private List<TppAccountAccess> fillAccountAccesses() {
+    List<Account> accountList = testDataService.getAccountsForBankOfferedConsent();
+    List<TppAccountAccess> accountAccessList = new ArrayList<>();
```

Comment:
this can be done without creating a local instance but directly returning it.

Reply:
doesn't work